### DISCUSSION TITLE
Site Not working on Github Pages

### DISCUSSION THREAD

**Discussion by rupeshtiwari** [mmistakes/minimal-mistakes#3025] [Q&A] [answered]
## Issue 
Site is not working on github pages it works on locally! 
Please let me know what mistake I am doing any help is appreciated.

## Environment
- Minimal Mistakes version: remote theme
- Ruby gem or remote theme version:  3.0
- Jekyll version:  4.2 
- Git repository URL:https://github.com/rupeshtiwari/rupeshtiwari.github.io
- GitHub Pages hosted (if yes provide URL to site): rupeshtiwari.com
- Operating system: windows

## Expected behavior
show work both in local and ghpages

## Steps to reproduce the behavior
I installed  the theme, customized _config.yml, added your own posts, and started up a 
  Jekyll server locally.

**Comment by rupeshtiwari** [comment, marked as answer]
This is fixed now.
I had to force deploy my build to github pages. 
Any one need help try this to force build in github
```
#!/bin/bash
curl -u rupeshtiwari:GH_token -X POST https://api.github.com/repos/rupeshtiwari/rupeshtiwari.github.io/pages/builds
```

**Comment by rupeshtiwari** [comment]
now site is up thanks

**Comment by cresusjpt** [comment]
> 
> 
> This is fixed now.
> I had to force deploy my build to github pages.
> Any one need help try this to force build in github
> 
> ```
> #!/bin/bash
> curl -u rupeshtiwari:GH_token -X POST https://api.github.com/repos/rupeshtiwari/rupeshtiwari.github.io/pages/builds
> ```

Hello @rupeshtiwari  can you please help me fix my issue. i have the same problem. my blog is correct in local but when i push on github the theme doesn't apply.
